plugins/MauticCrmBundle/Api/SalesforceApi.php

```diff
@@ -15,6 +15,9 @@ class SalesforceApi extends CrmApi
     protected $requestSettings = [
         'encode_parameters' => 'json',
     ];
+
+    const CURL_SSLVERSION_TLSv1_1 = 1;
```

Comment:
The tests are still failing. My explanation is that __construct is called before the const is being declared. I see the problem on line 28 where it is asking if the `CURL_SSLVERSION_TLSv1_1` constant is defined and using it either way. It should set a default value if the const is not defined instead.